cmd/liqonet/route-operator.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+
+	routeoperator "github.com/liqotech/liqo/internal/liqonet/route-operator"
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
+	"github.com/liqotech/liqo/pkg/liqonet/overlay"
+	liqorouting "github.com/liqotech/liqo/pkg/liqonet/routing"
+	"github.com/liqotech/liqo/pkg/liqonet/utils"
+	"github.com/liqotech/liqo/pkg/mapperUtils"
+)
+
+type routeOperatorFlags struct {
+	vni      int
+	Mtu      int
+	VtepPort int
+}
```

Comment:
Could be defined as global variables to avoid moving back and forth those structs. Additionally, it should be made consistent whether they are exported or not

Reply:
Totally agree on the consistency. I personally prefer to have a struct that encapsulates the flags, but no strong opinions on it.

Reply:
Perfectly fine from my side